services/qan/client.go

```diff
@@ -516,4 +517,24 @@ func fillPostgreSQL(mb *qanpb.MetricsBucket, bp *agentpb.MetricsBucket_PostgreSQ
 	mb.ApplicationName = bp.ApplicationName
 	mb.Planid = bp.Planid
 	mb.QueryPlan = bp.QueryPlan
+	mb.HistogramItems = convertHistogramItems(bp.HistogramItems)
+}
+
+func convertHistogramItems(items []*agentpb.HistogramItem) []string {
+	res := []string{}
+	for _, v := range items {
+		item := &qanpb.HistogramItem{
+			Range:     v.Range,
+			Frequency: v.Frequency,
+		}
+
+		json, err := json.Marshal(item)
+		if err != nil {
+			continue
```

Comment:
I can pass *Client to fillPostgreSQL and convertHistogramItems if it is needed to show warning or error here. In any other fill func we dont doing this.